gcsio/src/main/java/com/google/cloud/hadoop/gcsio/StorageStubProvider.java

```diff
@@ -51,7 +51,7 @@ public StorageStub buildAsyncStub() {
   }
 
   private Map<String, Object> getGrpcServiceConfig() {
-    Map<String, Object> name = ImmutableMap.of("service", "google.storage.v1.Storage");
+    Map<String, Object> name = ImmutableMap.of("service", readOptions.getGrpcServiceName());
```

Comment:
I think that we should override this instead:
https://github.com/GoogleCloudDataproc/hadoop-connectors/blob/630a99d2c78bfb1d7647d11af8381f3e1e757093/gcsio/src/main/java/com/google/cloud/hadoop/gcsio/StorageStubProvider.java#L40

Reply:
updated the property to be hostname.